src/components/create-courses/component/authors-lists.jsx

```diff
@@ -0,0 +1,71 @@
+import React, {useMemo} from "react";
+import { Button, List, Col, Row } from "reactstrap";
+
+export default function AuthorsLists({ authorsList, setAuthorsList }) {
+
+	const authorsNotSelected = useMemo( () => {
+		return authorsList.filter(({ selected }) => !selected);
+	}, [authorsList])
+
+	const authorsSelected = useMemo( () => {
+		return authorsList.filter(({ selected }) => selected);
+	}, [authorsList])
+
+	function selectToggle(authorId) {
+		const newArray = authorsList.filter((author) => author.id !== authorId);
```

Comment:
Let's use just map to reduce amount of operations

Reply:
Done!